docs/en/enterprise-edition/policy-reference/aws-policies/aws-iam-policies/bc-aws-iam-46.adoc

```diff
@@ -28,75 +27,37 @@
 
 === Description 
 
+This policy ensures that AWS SQS policies do not allow unrestricted (*) actions. Allowing all actions in a policy can expose your SQS queue to potential misuse or unauthorized access, posing a security risk. Restricting the actions to only those required for your application ensures adherence to the principle of least privilege.
```

Comment:
This policy ensures that AWS SQS policies are configured to limit permissions to specific actions, avoiding the use of unrestricted wildcards like (*), in adherence to the principle of least privilege. By restricting actions to only those necessary for your application, you mitigate the risk of unauthorized access and potential misuse of your SQS queue.